integration-libs/cdc/components/login/facade/index.ts

```diff
@@ -0,0 +1 @@
+export * from './cdc-login-form-component.service';
```

Comment:
```suggestion
export * from './cdc-login-form-component.service';
export * from './cdc-login-form.module';
```

let's expose in public API also the module